docs/src/pages/quasar-cli/quick-start.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Quick Start
+desc: Get up and developing a Quasar app in less than 3 minutes.
+---
+
+It takes three steps and in a couple of minute, you are off a running with a full-fledged Vue app, built with state-of-the-art best practices via Quasar's CLI. 
```

Comment:
Even with the suggestion, something sounds off. Maybe a side effect of multiple quick iterations? Does it sound right?
```suggestion
It takes three steps and in a couple of minutes, you are off a running with a full-fledged Vue app, built with state-of-the-art best practices via Quasar's CLI. 
```